### ISSUE TITLE
[4.1] Field with name `file` breaks 

### ISSUE THREAD

**Issue by lucaguindani** [Laravel-Backpack/CRUD#2784]
# Bug report

### What I did

Update Backpack from 4.0 to 4.1.

### What happened

I get the following error when I try to create or edit a record.

```
file() expects parameter 1 to be a valid path, object given (View: /var/www/vendor/backpack/crud/src/resources/views/crud/fields/inc/wrapper_start.blade.php)
```
```
Illuminate\Foundation\Bootstrap\HandleExceptions::handleError
vendor/backpack/crud/src/resources/views/crud/fields/inc/wrapper_start.blade.php:7
```

I tracked the issue to the "upload" field below:

```
CRUD::addField([
    'name' => 'file',
    'label' => 'Document',
    'type' => 'upload',
    'upload' => true,
    'tab' => 'Champs d\'administration'
]);
```

If I delete the field everything works again. This wasn't an issue with Backpack 4.0 and the field parameters haven't changed in 4.1 (https://backpackforlaravel.com/docs/4.1/crud-fields#upload).

### Versions

PHP 7.4
Laravel 7.10.3

**Comment by pxpm** [comment]
Hello @lucaguindani 

I am not sure how we should tackle this one yet. 

I found the culprit in `upload.blade.php`: 

```php
    if (!isset($field['wrapper']) || !isset($field['wrapper']['data-field-name'])) {
        $field['wrapper']['data-field-name'] = $field['name'];
    }
```
This will trigger the error because: ` is_callable('file')` returns true. I guess that is because `file` is actually a php function. 

One way of solving this would be if you setup in your field: 

```php
CRUD::addField([
    'name' => 'file',
    'label' => 'Document',
    'type' => 'upload',
    'upload' => true,
    'tab' => 'Champs d\'administration',
    'wrapper' => [
    'data-field-name' => 'my_custom_field_name_to_dont_throw_error'
    ]
]);
```
I think it will not break the field functionality atleast in my first test run, could you give it a try ? 

Best,

**Comment by lucaguindani** [comment]
> One way of solving this would be if you setup in your field:
> 
> ```
> CRUD::addField([
>     'name' => 'file',
>     'label' => 'Document',
>     'type' => 'upload',
>     'upload' => true,
>     'tab' => 'Champs d\'administration',
>     'wrapper' => [
>     'data-field-name' => 'my_custom_field_name_to_dont_throw_error'
>     ]
> ]);
> ```
> 
> I think it will not break the field functionality atleast in my first test run, could you give it a try ?

Hello @pxpm, I tried your "wrapper" workaround and the field is working again as expected.

**Comment by pxpm** [comment]
I'v sent a PR to fix this #2858 . Lets keep any discussion in PR thread!. 

Thanks again @lucaguindani for spotting this issue. 

Best to you!
